interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -171,17 +210,41 @@ static int getPort() {
   protected TestServiceGrpc.TestServiceBlockingStub blockingStub;
   protected TestServiceGrpc.TestServiceStub asyncStub;
 
+  private final LinkedBlockingQueue<ClientStreamTracer> clientStreamTracers =
+      new LinkedBlockingQueue<ClientStreamTracer>();
+
+  private final ClientStreamTracer.Factory clientStreamTracerFactory =
```

Comment:
I've not really been happy with how the interop tests have been impacted by stats testing. I feel like its abusive of the tests and making them drift from their purpose (high level, "does this feature work" from the user's point of view). And all this boilerplate is making it worse. I'm okay with this going in, but we need to come up with a longer-term plan.